### DISCUSSION TITLE
Make "x-frame options" configurable

### DISCUSSION THREAD

**Discussion by bbarou** [ampache/ampache#3179] [Ideas]
Release 5 added the header "X-Frame-Options: SAMEORIGIN" on the login page.
I use ampache with Organizr and Ampache can no longer be framed if not logged in beforehand.
Ideally there would be an option in the config file to configure the header to allow one's domain with the default being "SAMEORIGIN".
what do you think?

**Comment by lachlan-00** [comment]
I'm going to make that change this week for the next release

**Reply by lachlan-00** [reply]
https://github.com/ampache/ampache/issues/3199
